config/example/config.toml

```diff
@@ -321,7 +321,7 @@ starlight = true
 # Disable lobby music on the server
 disable_lobby_music = false
 # Enable this if you want to disable the popup alert for people on the same CID
-disable_cid_warning_popup = false
+disable_cid_warning_popup = true
```

Comment:
Don't worry about the codeowner review AA, this is enabled just for local testing